### PR TITLE
Add op_vault-round-2-op_ctv review to bitcoin-review-podcast

### PR DESCRIPTION
**Pull request by markon1-a**
This PR adds [op_vault-round-2-op_ctv](https://www.youtube.com/watch?v=h4ReIIapN8Y) transcript review to the bitcoin-review-podcast directory.

### CODE REVIEW

**Review by kouloumos — COMMENTED** [review]
Thank you for your contribution @markon1-a! The work that you put into this is visible.

I understand that is not easy to catch all the technical jargon that is discussed in those transcripts - we are working on making it easier. I left a bunch of comments inline that need to be resolved before merging this. 

Thank you for your patience and for helping us making this experience better for all.

File: bitcoin-review-podcast/op_vault-round-2-op_ctv.md [hunk 1]
```diff
@@ -86,7 +90,7 @@ And so for a long time, there's been this idea floating around of vaults.
 I think the earliest mention I could find was 2013 in some Bitcoin talk forums.
 And the basic idea with vaults is you can lock up your coins in such a way that they're still spendable by you, of course, but if you want to spend them somewhere, you basically say beforehand, okay, when I spend this coin, I'm going to wait some period, whether that's 10 blocks or two days, whatever it might be.
 And you publicly declare the intent to spend that coin to some particular destination.
-And during that period, you can come in and say, oh, wait a second, I don't recognize that spend, or I didn't mean to do that spend, or NVK backdoored my hardware wallet, and so now I want to recover those coins or claw them back or cancel that transaction.
+And during that period, you can come in and say, oh, wait a second, I don't recognize that spend, or I didn't mean to do that spend, or `NVK` backdoored my hardware wallet, and so now I want to recover those coins or claw them back or cancel that transaction.
```
[the remark this answers is not in the document]
Comment (markon1-a) [reply bitcoin-review-podcast/op_vault-round-2-op_ctv.md:94]: Here I thought NVK is a term and not a name of a speaker.

**Comment by kouloumos** [comment]
@markon1-a is this ready for review? Because I explicitly mentioned that for words like `OP_VAULT` and `CTV` I only gave some examples that needed to change and I didn't comment on every one.

File: bitcoin-review-podcast/op_vault-round-2-op_ctv.md [hunk 2]
```diff
@@ -97,74 +101,72 @@ You have to worry about fee management.
...
+So like when we start the withdrawal process for this vault, which wallet are the funds actually going to flow into on their way to the final destination where you actually kind of want to put the Bitcoin, you know, whether that's sending it to somebody or uploading to an exchange or whatever.
 So you pick the various parameters that are involved.
 And then what you have to do to actually get this behavior in Bitcoin today is you have to decide on what all the possible flows are of funds.
-So that might be, OK, I want to create a path where I'm putting in 10 Bitcoin to my vault, I wish, right?
-And then I'm going to allow, you know, a withdrawal of five Bitcoin at a time, or I'm going to allow a withdrawal of, you know, one Bitcoin at a time.
+So that might be, I want to create a path where I'm putting in 10 Bitcoin to my vault, I wish, right?
+And then I'm going to allow, a withdrawal of five Bitcoin at a time, or I'm going to allow a withdrawal of, one Bitcoin at a time.
```
[the remark this answers is not in the document]
Comment (markon1-a) [reply bitcoin-review-podcast/op_vault-round-2-op_ctv.md:135]: Yes thanks I will.

File: bitcoin-review-podcast/op_vault-round-2-op_ctv.md [hunk 3]
```diff
@@ -1,68 +1,74 @@
...
+categories: ["podcast"]
 date: 2023-05-11
 ---
 ## Introductions
 
-Speaker 0: 00:01:25
+NVK: 00:01:25
 
-Today, we're going to get back to OBVault, just kidding, UpVault, this new awesome proposal on how we can make people's money safe in the future.
+Today, we're going to get back to opvault, just kidding, opvault, this new awesome proposal on how we can make people's money safe in the future.
```
[the remark this answers is not in the document]
Comment (markon1-a) [reply bitcoin-review-podcast/op_vault-round-2-op_ctv.md]: Have changed them.

File: bitcoin-review-podcast/op_vault-round-2-op_ctv.md [hunk 4]
```diff
@@ -97,74 +101,72 @@ You have to worry about fee management.
 You have to worry about fixing addresses that the coins ultimately flow into.
 So if you're a really, really big company, maybe you can do this and maybe you can eke out some security benefits.
 But if you're an individual or you know kind of a smaller operation it's really not as practical.
-So I've been kind of involved in various custody efforts including my own for a while and I've been thinking about how to use vaults and I did like a very simple implementation using OPCHECK template verify And this makes vaults kind of more achievable, but there were still some downsides.
-Basically, OPTCHECK template verify is just a way of saying, hey, we're gonna lock these coins up and they're spendable into this particular set of outputs with like no signature or anything else.
+So I've been kind of involved in various custody efforts including my own for a while and I've been thinking about how to use vaults and I did like a very simple implementation using `OPCHECK` template verify. And this makes vaults kind of more achievable, but there were still some downsides.
```
[the remark this answers is not in the document]
Comment (markon1-a) [reply bitcoin-review-podcast/op_vault-round-2-op_ctv.md]: Should I also backtick CTV?

File: bitcoin-review-podcast/op_vault-round-2-op_ctv.md [hunk 5]
```diff
@@ -945,93 +922,92 @@ So you're using, I think it's something on the order of like 32 more bytes to do
...
 ## Use in chaumian mints/ecash
 
-Speaker 5: 00:52:42
+Rijndael: 00:52:42
 
-So NVK, you just mentioned Cashew, you know, We just talked about Chami and Mints not that long ago.
-This would actually also be great to add to Chami and Mints, right?
-So if you have either a Feti Mint or a Cashew Mint and you want to say, all right, we're going to have our Mint's treasury in a vault that has like a, you know, this time locked predestined withdrawal path so that if either one of the functionaries of the mint or if somebody pwns the software and tries to like run away with the money, then we can claw it back.
-This fits in really, really well with the idea that maybe we'll have some collaborative custodyments for a lot of users.
+So NVK, you just mentioned Cashew, we just talked about `Chaumi` and `Mints` not that long ago.
```
[the remark this answers is not in the document]
Comment (kouloumos) [reply bitcoin-review-podcast/op_vault-round-2-op_ctv.md]: You got this, but the same mistake exists one line below and you haven't handle it.

File: bitcoin-review-podcast/op_vault-round-2-op_ctv.md [hunk 6]
```diff
@@ -1254,137 +1204,130 @@ And these fuckers just keep on transferring to each other without transferring t
...
+So they're going back to, to reinsurers and, all the underwriters and everybody sort of say, hey, you know, this is how we think we could do it so that the coins are actually safe, can insure this, right?
 So they're trying to find the packages that meet the technology that can be done.
-And they're sort of like a mini script sort of makes their life easier to, yeah, no, nothing to do with ephemeral and CoreWatch.
+And they're sort of like a Miniscript sort of makes their life easier to, yeah, no, nothing to do with ephemeral and CoreWatch.
 So they're trying to just create policies based on setups.
 And if you want to get your ear talked off, just go to ask Rob about Miniscript.
 
 ## Now really, how do we activate OP_VAULT??
 
-Speaker 0: 01:08:56
```
Comment (kouloumos) [inline bitcoin-review-podcast/op_vault-round-2-op_ctv.md:1365]: There was no need to delete this timestamp.

**Comment by kouloumos** [comment]
I believe the transcript is now ready to merge. Thank you for your efforts.

For future reviews, I recommend considering the comments I have made in previous reviews. When these comments are not addressed, it becomes challenging for me to evaluate your changes, as it impacts my trust in the modifications. This, in turn, extends the evaluation time.

Additionally, I suggest choosing topics that genuinely interest you and that you are eager to read about. From your recent reviews, it seems that there might be a lack of context regarding the discussed topics. To ensure high-quality work, it is crucial to engage with the content deeply. These transcripts cover intricate Bitcoin technical topics, and a genuine interest in these areas will significantly enhance your reviews.

Thank you for your attention to these points, and I look forward to your future contributions.
